sycl/include/CL/sycl/detail/pi.h

```diff
@@ -505,6 +508,15 @@ constexpr pi_sampler_properties PI_SAMPLER_PROPERTIES_ADDRESSING_MODE =
 constexpr pi_sampler_properties PI_SAMPLER_PROPERTIES_FILTER_MODE =
     CL_SAMPLER_FILTER_MODE;
 
+using pi_memory_order_capabilities = pi_bitfield;
+typedef enum {
+  PI_MEMORY_ORDER_RELAXED = 0x01,
+  PI_MEMORY_ORDER_ACQUIRE = 0x02,
+  PI_MEMORY_ORDER_RELEASE = 0x04,
+  PI_MEMORY_ORDER_ACQ_REL = 0x08,
+  PI_MEMORY_ORDER_SEQ_CST = 0x10,
+} _pi_memory_order_capability;
```

Comment:
The idea is that the typenames starting with "_pi" are internal to this header, and exposed as corresponding "using pi_ =". In this case either remove leading "_", or follow the lines 503-509

Reply:
My bad! I think I confused myself with the enum. I have gone with the suggestion of following the bitfield in 503-509. Thank you!